src/HotChocolate/Fusion/src/Fusion.Execution/HotChocolate.Fusion.Execution.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>HotChocolate.Fusion.Planning</AssemblyName>
+    <RootNamespace>HotChocolate.Fusion</RootNamespace>
+    <Nullable>enable</Nullable>
```

Comment:
Can be removed.

Reply:
Well spotted! I renamed the project.

Reply:
y u merge with unresolved conversations? 😐